### ISSUE TITLE
[MGPG-116] Up the file size limit to 64K

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-gpg-plugin#166]
**[Tamas Cservenak](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=cstamas)** opened **[MGPG-116](https://issues.apache.org/jira/browse/MGPG-116?redirect=false)** and commented

The refd document talks about "key size" (that is very rarely 16K), but the file we are loading up is armored: Base64 encoded + formatted  + newlines and may have arbitrary amount of comments added as well.

In short, the wiki talks about the "key size" as in memory, while this file have much of the overhead, plus, it may be collection of keys...

---

**Remote Links:**
- [GitHub Pull Request #85
](https://github.com/apache/maven-gpg-plugin/pull/85)
